### ISSUE TITLE
Using a different optimizer in pipeline config file

### ISSUE THREAD

**Issue by waseemR02** [EdjeElectronics/TensorFlow-Lite-Object-Detection-on-Android-and-Raspberry-Pi#169]
As far as I understand, I can only use the `optimizers` that are defined in `optimizer.proto` which are namely `RMSProp`, `Adam` and `Momentum`. And I can also tamper with their learning rate and other such params. 
How do I use some `meta-heuristic`optimizer in the config file? Because I could probably use some library for that but how do I about using config file? And if I can't, do i have to scrape the whole config file idea and do it the usual way?
Any help would be appreciated. Thank you
